pom.xml

```diff
@@ -34,9 +34,9 @@
 
     <scm>
         <connection>scm:git:https://github.com/openrewrite/rewrite-maven-plugin.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/openrewrite/rewrite-maven-plugin.git</developerConnection>
+        <developerConnection>scm:git:https://github.com/openrewrite/rewrite-maven-plugin.git</developerConnection>
```

Comment:
this is to allow for a `git push` to occur using a `{{github.token}}`. A token works with https, whereas `ssh` requires an actual ssh key. The tradeoff is we get a `github.token` for free in github actions, but if a _human_ were to try to manually do this on their local computer, they'd need to have a github token for auth rather than using what I think most people use-- an ssh key. It's not a huge deal, it's just something to note in case there's a need for manual intervention. 

I could see if the `scm` can be modifiable under a `profile` to use `https` for `release` and `ssh` for everything else